### DISCUSSION TITLE
AsyncAPI with SQS

### DISCUSSION THREAD

**Discussion by xescuder** [microcks/microcks#770] [Community]
Hi. I want to use Microcks with Amazon SQS (I've AsyncAPI defined), but I don't see how. Anyone can help me?

Thanks!

**Comment by lbroudoux** [comment]
Hi there!

SQS is not on the list of supported protocols/bindings for AsyncAPI at the moment. That said we regularly add new protocols based on demand from the community and if get find a "sponsor" for helping test the new connector. Are you making heavy usage of SQS? Would you be interested in?

**Reply by xescuder** [reply]
Hi, yes. We're making heavy usage of SQS. We can help to test the new connector.

**Reply by yada** [reply]
For the ones who are following this post, this request from @xescuder is WIP ;-) 
Let us know if you also need SQS integration.

**Comment by lbroudoux** [comment]
Hey there! This one is now planned in microcks/microcks#793 😉

**Reply by lbroudoux** [reply]
And it is now released as part of the `1.7.1` release! See the full guide here: https://microcks.io/documentation/guides/aws-sqs-sns-support/
